docs/pages/guides/keyboard-handling.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Keyboard handling
+description: A guide for handling common keyboard interactions.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+Keyboard handling is crucial for creating an excellent user experience in your Expo app. This guide covers common keyboard interactions and how to manage them effectively.
+
+React Native provides several keyboard-related APIs, with [`Keyboard`](https://reactnative.dev/docs/keyboard) and [`KeyboardAvoidingView`](https://reactnative.dev/docs/keyboardavoidingview) being the most frequently used. For more complex or custom keyboard interactions, you might consider using [Keyboard Controller](https://kirillzyusko.github.io/react-native-keyboard-controller), which is a library that offers advanced keyboard handling capabilities.
+
+## The basics
+
+### Keyboard events
```

Comment:
I feel like the keyboard events are a less-frequent / more advanced use case than simple keyboard avoidance, so what if we put keyboard avoidance first and the events second?